packages/maskbook/src/plugins/Trader/UI/balancer/TradeRoute.tsx

```diff
@@ -0,0 +1,151 @@
+import classNames from 'classnames'
+import {
+    Box,
+    Breadcrumbs,
+    Typography,
+    makeStyles,
+    createStyles,
+    Paper,
+    AvatarGroup,
+    TableContainer,
+    Table,
+    TableHead,
+    TableRow,
+    TableCell,
+    TableBody,
+    Link,
+    Button,
+    ButtonBase,
+    Chip,
+} from '@material-ui/core'
+import NavigateNextIcon from '@material-ui/icons/NavigateNext'
+import { TokenIcon } from '../../../../extension/options-page/DashboardComponents/TokenIcon'
+import { useStylesExtends } from '../../../../components/custom-ui-helper'
+import { SwapResponse, TradeComputed, TradeProvider } from '../../types'
+import { formatEthereumAddress, formatPercentage } from '../../../Wallet/formatter'
+import { resolveTradePairLink } from '../../pipes'
+import BigNumber from 'bignumber.js'
+import { useI18N } from '../../../../utils/i18n-next-ui'
+
+const useStyles = makeStyles((theme) =>
+    createStyles({
+        root: {
+            '&::-webkit-scrollbar': {
+                display: 'none',
+            },
+            width: '100%',
+            boxSizing: 'border-box',
+            padding: theme.spacing(1.5, 2),
+            margin: theme.spacing(0, 'auto', 2),
+        },
+        table: {},
+        head: {
+            fontWeight: 300,
+            color: theme.palette.text.secondary,
+        },
+        cell: {
+            border: 'none',
+            padding: 0,
+        },
+        list: {
+            alignItems: 'center',
+            justifyContent: 'flex-start',
+        },
+        item: {
+            display: 'flex',
+            padding: theme.spacing(0.25, 0),
+        },
+        button: {
+            display: 'flex',
+            borderRadius: 500,
+            padding: theme.spacing(0.25),
+            '&:hover': {
+                backgroundColor: theme.palette.background.default,
+            },
+        },
+        link: {
+            display: 'flex',
+            alignItems: 'center',
+        },
+        name: {
+            fontSize: 12,
+            marginLeft: theme.spacing(1),
+        },
+    }),
+)
+
+export interface TradeRouteProps extends withClasses<KeysInferFromUseStyles<typeof useStyles>> {
```

Comment:
Please stop using `KeysInferFromUseStyles` in new code. It's deprecated.

Reply:
Removed!